src/cargo/ops/registry.rs

```diff
@@ -380,6 +380,19 @@ pub fn search(query: &str,
         };
         try!(config.shell().say(line, BLACK));
     }
+    if total_crates > limit as u32 && limit < 100 {
```

Comment:
Could this 100 constant be refactored to not be duplicated amongst a few locations?